source/OutfitterPanel.cpp

```diff
@@ -278,13 +277,6 @@ double OutfitterPanel::ButtonPanelHeight() const
 
 
 
-int OutfitterPanel::DetailWidth() const
```

Comment:
This had no uses.  Discovered while looking for simplistic functions that only called other functions.